docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,417 @@
+  named `antrea/antrea-multicluster-controller:latest` locally.
+2. Run `docker save antrea/antrea-multicluster-controller:latest > antrea-mcs.tar` to save the image.
+3. Copy the image file `antrea-mcs.tar` to the nodes of your local clusters
+4. Run `docker load < antrea-mcs.tar` in each node of your local clusters.
+
+## Install Mulit-Cluster Controller
+
+In a ClusterSet, there is one leader cluster and two or more member clusters. We allow
+user to run the leader controller in a dedicated cluster or a shared cluster. You can
+refer to [Installation in Dedicated Leader Cluster](#installation-in-dedicated-leader-cluster)
+and [Installation in the Member Cluster](#installation-in-the-member-cluster)
+to deploy leader and member controllers in separate clusters, or you can refer to
+[Installation in a Shared Cluster](#installation-in-a-shared-cluster) to learn how to
+run both member and leader controllers in a shared cluster.
+
+### Installation in Dedicated Leader Cluster
+
+1. Run below commands to apply global CRDs in the leader cluster.
+
+   * For any given release `<TAG>` (e.g. `v1.5.0`).
+
+     ```bash
+     kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-multicluster-leader-global.yml
+     ```
+
+   * To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+
+     ```bash
+     kubectl apply -f multicluster/build/yamls/antrea-multicluster-leader-global.yml
+     ```
+
+2. Install multi-cluster controller in the leader cluster, since multi-cluster controller is
+running as namespaced deployment, you should create a namespace first, then apply the
+manifest with new namespace. below are sample commands.
+
+* For any given release `<TAG>` (e.g. `v1.5.0`).
+  
+  ```bash
+  kubectl create ns antrea-mcs-ns
+  curl -L https://github.com/antrea-io/antrea/releases/download/v1.5.0/antrea-multicluster-leader-namespaced.yml >   antrea-multicluster-leader-namespaced.yml
+  sed 's/changeme/antrea-mcs-ns/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
+  ```
+  
+* To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+  
+  ```bash
+  kubectl create ns antrea-mcs-ns
+  multicluster/hack/generate-manifest.sh -l antrea-mcs-ns | kubectl apply -f -
+  ```
+  
+### Installation in the Member Cluster
+
+You can run below commands to install multi-cluster controller to all
+member clusters, it will install a member controller in the namespace `kube-system`
+by default.
+
+* For any given release `<TAG>` (e.g. `v1.5.0`).
+
+```bash
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-multicluster-member.yml
+```
+
+* To deploy the latest version (checkout and built from the `feature/multi-cluster` branch).
+
+```bash
+kubectl apply -f multicluster/build/yamls/antrea-multicluster-member.yml
+```
+
+### Installation in a Shared Cluster
```

Comment:
"shared cluster" is not the right term. Maybe "Leader and Member in one Cluster"?